tests/frontend/test_ratings.py

```diff
@@ -502,3 +504,151 @@ def test_delete_developer_reply_to_review(selenium, base_url, variables):
+        page.wait_for_page_to_load()
+        # if there are reviews
+        if page.rating_card.number_of_reviews_with_specific_stars(count) > 0:
+            # verify that each review in the list has the wanted number of stars
+            for review in review_page.review_items:
+                rating = int(review.rating_stars.text.split()[1])
+                assert 5 - count == rating
+
+
+@pytest.mark.nondestructive
+def test_rating_card_bar_review_counter(selenium, base_url, variables):
+    # this test verifies that the counter for each bar is correct
+    selenium.get(variables["addon_version_page_url"])
+    page = Versions(selenium, base_url)
+    for i in range(5):
+        counter_from_card = page.rating_card.number_of_reviews_with_specific_stars(i)
+        review_page = page.rating_card.click_see_all_reviews_with_specific_stars(i)
+        page.wait_for_page_to_load()
+        assert len(review_page.review_items) == counter_from_card
+        page.driver.back()
+
+
+@pytest.mark.nondestructive
+def test_rating_card_average_stars(selenium, base_url, variables):
+    # this test calculates the rating average and verifies that the displayed average is correct
+    selenium.get(variables["addon_version_page_url"])
+    page = Versions(selenium, base_url)
+    total_reviews_number = 0
+    total_stars_number = 0
+    for i in range(5):
+        total_reviews_number += page.rating_card.number_of_reviews_with_specific_stars(
+            i
+        )
+        # multiply the number of reviews with the star value of the reviews and add it to the total
+        total_stars_number += page.rating_card.number_of_reviews_with_specific_stars(
+            i
+        ) * (5 - i)
+    assert total_stars_number / total_reviews_number == page.rating_card.rating
+
+
+@pytest.mark.nondestructive
+def test_meta_card_loaded_correctly(selenium, base_url, variables):
+    selenium.get(f'{base_url}/addon/{variables["addon_with_stats"]}')
+    page = Detail(selenium, base_url)
+    meta_card = page.stats
+    assert meta_card.addon_user_stats.is_displayed()
+    assert meta_card.addon_reviews_stats.is_displayed()
+    assert meta_card.addon_star_rating_stats.is_displayed()
+    for i in range(5):
+        assert meta_card.bar_grouped_ratings[i].is_displayed()
+        assert meta_card.rating_bars[i].is_displayed()
+        assert meta_card.bar_rating_counts[i].is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_meta_card_reviews_counter(selenium, base_url, variables):
+    # this test verifies that the total number of reviews is equal to the sum of one-star to five-star reviews
+    selenium.get(f'{base_url}/addon/{variables["addon_with_stats"]}')
+    page = Detail(selenium, base_url)
+    reviews_sum = 0
+    for el in page.stats.bar_rating_counts:
+        reviews_sum += int(el.text.split()[0])
+    assert int(page.stats.addon_reviews_stats.text.split()[0]) == reviews_sum
```

Comment:
Are these two tests covering the stats card from the add-on detail page? If so, we already have tests for that section in the `test_addon_detail` suite. Check those tests to see if your scenarios are covered already. If not, add these tests to that suite.

Reply:
They are already covered, I deleted them